base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
html must be properly formatted. Like <head> tags must be tab level down. Same for all others in the page.